images/dcos/root/usr/html/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
why do we need this file? if we really need something here can we rip out this boiler-plate and replace it with something minimal, e.g. `<html><body>k8s/mesos master proxy</body></html>`